green_points:
- x: 2.222071409225464
  y: 0.26930251717567444
- x: 4.04038143157959
  y: 0.7433702349662781
- x: 3.5627281665802
  y: -1.9975024461746216
- x: 1.1601217985153198
  y: 1.3191025257110596
- x: 2.6555328369140625
  y: 0.21301133930683136
- x: 4.277427673339844
  y: -0.10996543616056442
- x: 1.9267034530639648
  y: -1.8563321828842163
red_points:
- x: 3.242326498031616
  y: -0.45878615975379944
- x: 3.327350616455078
  y: 0.9397451281547546
- x: 2.1248772144317627
  y: 1.1365119218826294
- x: 1.3408622741699219
  y: 0.00427410751581192
- x: 2.514784812927246
  y: -0.9197416305541992
- x: 2.5286014080047607
  y: -1.1295424699783325
- x: 2.6685314178466797
  y: -1.017767071723938
